lib/activerecord_spanner_adapter/base.rb

```diff
@@ -4,6 +4,8 @@
 # license that can be found in the LICENSE file or at
 # https://opensource.org/licenses/MIT.
 
+require_relative "relation"
```

Comment:
Prefer absolute rather than relative requires (i.e. `require "activerecord_spanner_adapter/relation"`)

Reply:
Done (but not for the samples where the relative requires refer to the sample configuration)